src/backend/columnar/columnar_customscan.c

```diff
@@ -535,7 +541,11 @@ ColumnarIndexScanAdditionalCost(PlannerInfo *root, RelOptInfo *rel,
 	 * "anti-correlated" (-1) since both help us avoiding from reading the
 	 * same stripe again and again.
 	 */
+#if PG_VERSION_NUM >= PG_VERSION_16
+	double absIndexCorrelation = fabs(indexCorrelation);
```

Comment:
Can we have something like this in compat.h file?

```
#if PG_VERSION_NUM >= PG_VERSION_16
             #define float_abs(a) fabs(a)
#else 
             #define  float_abs(a) abs(a) 
```

and change the below to

`double absIndexCorrelation = float_abs(indexCorrelation);`